src/util/FactorUtil.js

```diff
@@ -24,69 +24,69 @@ function (Okta, TimeUtil) {
   var factorData = {
     'OKTA_VERIFY': {
       label: 'factor.totpSoft.oktaVerify',
-      description: Okta.loc('factor.totpSoft.description', 'login'),
+      description: _.partial(Okta.loc, 'factor.totpSoft.description', 'login'),
```

Comment:
might be easier to do this the same way we do `label` one line above. We have the key here and we handle this properly in the function `getFactorLabel`.